src/components/core/DpEditor/DpEditor.vue

```diff
@@ -477,16 +477,16 @@ export default {
         const suggestionGroupSchema = {
           matcher: {
             char: '@|$|#', // A single char that should trigger a suggestion
-            allowSpaces: true || false,
-            startOfLine: true || false
+            allowSpaces: 'boolean',
+            startOfLine: 'boolean'
```

Comment:
Here and lines 488-489: why are you changing this?

Reply:
damn'd. This was just a tryout because. SonarCloud mentioned something. It was not supposed to be in the PR. I revert it...
544581f3e9a979d7852f74d3f32d3627c8253168